src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -222,27 +224,85 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
                                   return false;
                                 });
 
-                                return (
-                                  <FilesTableRow
-                                    key={file.identifier}
-                                    file={file}
-                                    removeFile={() => {
-                                      const associatedArtifactsBeforeRemoval = associatedArtifacts.length;
-                                      const remainingFiles = uppy
-                                        .getFiles()
-                                        .filter((uppyFile) => uppyFile.response?.uploadURL !== file.identifier);
-                                      uppy.setState({ files: remainingFiles });
-                                      remove(associatedFileIndex);
+                                if (!file.administrativeAgreement) {
```

Comment:
Tror vi kan fjerne denne sjekken, og heller lage en ny `const filesToPublish` på linje 69, og bruke den for `map` på over her